Programmers/해시_폰켓몬/seungbinpark.py

```diff
@@ -0,0 +1,7 @@
+def solution(nums):
+    n = set(nums) # 중복 제외 종류 번호의 집합
+    N = len(nums)/2 # 이론상의 최댓값
+    # 종류 번호의 종류가 N/2개보다 적을 때
+    if N>=len(n): return len(n) 
+    # 종류 번호의 종류가 N/2개보다 많을 때
+    elif N<len(n): return N
```

Comment:
if 에서 `N >= len`을 고려하고 있으므로, elif 보다 else를 사용하는 게 더 효율적일 것 같다는 생각이 들었습니다!
set(nums)를 활용해서 한 번에 중복을 제외한 폰켓몬 종류의 수를 얻어내는 과정이 좋은 것 같아요!

Reply:
지윤님이 언급하신 부분처럼 이미 elif의 조건이 if문 조건과 반대되고, if문을 만족하면 return으로 값을 반환하기 때문에 elif 를 쓰지 않고 그냥 return만 써도 괜찮을 것 같아요!

```suggestion
    return N
```